packages/synapse-react-client/src/components/SynapseHomepageV2/SynapseSearchChips.tsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react'
+import { Chip, Typography } from '@mui/material'
+import {
+  SearchQuery,
+  KeyValue,
+  SearchFieldName,
+  SearchFacetSort,
+} from '@sage-bionetworks/synapse-types'
+
+const searchValues = [
+  "Alzheimer's Disease",
+  'Neurofibromatosis mouse models',
+  'Cancer',
+  'UK Biobank',
+  'Single-cell analysis of breast cancer',
+  'ROSMAP Religious Orders Study and Memory and Aging Project',
+  'Retinal regeneration studies',
+  'Machine Learning datasets',
+]
+
+const projectKeyValue: KeyValue = {
+  key: 'node_type',
+  value: 'project',
+  not: false,
+}
+
+export type SynapseSearchChipsProps = {
+  gotoPlace: (href: string) => void
+}
+
+export const SynapseSearchChips: React.FunctionComponent<
+  SynapseSearchChipsProps
+> = ({ gotoPlace }) => {
+  return (
+    <>
+      {searchValues.map(value => {
+        return (
+          <Chip
+            key={value}
+            label={
+              <Typography
+                variant="body1"
+                sx={{
+                  fontSize: '18px',
+                  fontWeight: 400,
+                  p: '10px',
+                }}
+              >
+                {value}
+              </Typography>
+            }
+            onClick={() => {
+              const searchQuery: SearchQuery = {
```

Comment:
Instead of just passing the search term, pass the SearchQuery json to the SWC Search place.  This is required for "cancer" term (since they want to filter to Projects only), but is useful for other search terms since we will no longer split the term by whitespace, which is appropriate for these targeted searches.